src/C1_verification.py

```diff
@@ -112,6 +112,7 @@ def all_valid_intervals(name, value, title):
         "lifetime": ["largerzero", "any"],
         "age_installed": [0, "any"],
         "installedCap": [0, "any"],
+        "maximumCap": [0, "any"],
```

Comment:
Actually, maximumCap can be [0,"any"] or None, right?

Reply:
yes.